examples/kubernetes.yml

```
@@ -38,6 +38,17 @@ spec:
           volumeMounts:
             - mountPath: /data
               name: data
+        livenessProbe:
+          exec:
+            command: 
+              - /bin/sh
+              - -c 
+              - '/mc-monitor status-bedrock --host 127.0.0.1 --port 19132 | grep online'
```

No need to grep the output since exit code already indicates healthiness, so can be called directly and default args are sufficient
```suggestion
              - /usr/local/bin/mc-monitor
              - status-bedrock
```